### PR TITLE
add network port check

### PR DESCRIPTION
**Pull request by miaojuncn**
add network port check

### CODE REVIEW

**Review by rainchen — CHANGES_REQUESTED** [review]
please clean up your commits, since they will be used as release notes

**Comment by rainchen** [comment]
@miaojun0 how about the dashboard? how to present this new metric? please upload the screenshot.
When adding/changing a new metric, don't forget to update the document at  docs/information_architecture.md

File: agent/.env.example [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
+NETWORK_PORT=4000
 
 # CITA node directory analysis
 NODE_DIR=/data/cita_secp256k1_sha3/test-chain/0
 SOFT_PATH=/data/cita_secp256k1_sha3/
-
-# CITA NODE INFO
-CITA_NODENAME=node_3
-CITA_CHAIN_ID=opc-devnet/3
-CITA_NETWORKPORT=4003
```
Comment (rainchen) [inline agent/.env.example]: are these vars useless? if so, should update the related documents

Comment (rainchen) [reply agent/.env.example]: should do these changes uisng a new PR

File: README-CN.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Summary Dashboard Demo
 ![summary-dashboard-demo-fs8](https://user-images.githubusercontent.com/71397/57682153-b9a5c700-7663-11e9-93c6-a29758e7d3a1.png)
 
 CITA Node Info Dashboard Demo
-![cita-node-info-dashboard-demo-fs8](https://user-images.githubusercontent.com/71397/57681838-15bc1b80-7663-11e9-91b4-202c306a0f3b.png)
+![cita-node-info-dashboard-demo-fs8](https://user-images.githubusercontent.com/20398034/106434137-18504b80-64ac-11eb-86be-f3c1133e7bd3.png)
```
Comment (rainchen) [inline README-CN.md]: should not use your local machine as example, use the testnet instead

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Summary Dashboard Demo
 ![summary-dashboard-demo-fs8](https://user-images.githubusercontent.com/71397/57682153-b9a5c700-7663-11e9-93c6-a29758e7d3a1.png)
 
 CITA Node Info Dashboard Demo
-![cita-node-info-dashboard-demo-fs8](https://user-images.githubusercontent.com/71397/57681838-15bc1b80-7663-11e9-91b4-202c306a0f3b.png)
+![cita-node-info-dashboard-demo-fs8](https://user-images.githubusercontent.com/20398034/106434137-18504b80-64ac-11eb-86be-f3c1133e7bd3.png)
```
Comment (rainchen) [inline README.md]: should not use your local machine as example, use the testnet instead

File: agent/.env.example (hunk 1 above)
Comment (rainchen) [reply agent/.env.example]: @miaojun0

File: agent/cita_exporter/cita_monitor_agent.py [hunk 4]
```diff
@@ -266,6 +272,14 @@ def exporter():
         return Response(prometheus_client.generate_latest(registry), mimetype="text/plain")
 
     service_status.labels(NodeIP=node_ip, NodePort=node_port).set(1)
+
+    network_port = int(os.getenv('NETWORK_PORT'))
```
Comment (rainchen) [inline agent/cita_exporter/cita_monitor_agent.py]: can't we get this via config? we already have this `SOFT_PATH`

Comment (miaojuncn) [reply agent/cita_exporter/cita_monitor_agent.py]: the port write in config file is container port, may not the port mapping to the host. i just follow the way of json rpc port

**Comment by miaojuncn** [comment]
WechatIMG2178

File: agent/README.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ Monitor Agent 是使用 `Python` 作为脚本语言，使用 `cita-cli` 工具
 ```
 cd agent
 cp .env.example .env
-修改 .env 配置文件中 HOSTNAME、NODE_IP、NODE_PORT、NODE_DIR、SOFT_PATH、CITA_NODENAME、CITA_CHAIN_ID、CITA_NETWORKPORT 为实际部署的信息
```
Comment (rainchen) [inline agent/README.md:20]: why removing `CITA_CHAIN_ID`? it is stilling using in `.env.example` as `CITA_CHAIN_ID=opc-devnet/3`

Comment (rainchen) [reply agent/README.md:20]: you're removing env variable, this is a breaking change, should update the commit message

Comment (miaojuncn) [reply agent/README.md:20]: These variables have not been used actually in previous versions. I revert changes temporarily，and edit these variables in next change.

File: agent/.env.example [hunk 6]
```diff
@@ -1,9 +1,10 @@
 # citamon_agent_cita_exporter HostName
 HOSTNAME=nodehostname
 
-# CITA node ip and port
+# CITA node ip, json_rpc port and network port
 NODE_IP=1.1.1.1
 NODE_PORT=1337
+NETWORK_PORT=4000
```
Comment (rainchen) [inline agent/.env.example]: use `NODE_` as prefix, should keep the same naming style as previous

**Review by rainchen — APPROVED** [review]

**Review by rainchen — APPROVED** [review]

**Review by rainchen — APPROVED** [review]

**Pull request merged**